core/templates/services/svg-sanitizer.service.ts

```diff
@@ -154,48 +154,74 @@ export class SvgSanitizerService {
     return { tags: invalidTags, attrs: invalidAttrs };
   }
 
+  // Returns SVG by parsing DataURI.
```

Comment:
This line might be confusing and can be removed. We have already defined the return type below.